vars/unitTestPost.groovy

```diff
@@ -25,7 +31,8 @@ def call(Map config = [:]) {
 
   def always_script = config.get('always_script',
                                  'ci/unit/test_post_always.sh')
-  sh script: always_script,
+  def valgrind = config.get('valgrind', '')
```

Comment:
Same as above, the defaults need to come from the Map stage_info that is populated a few lines down.